### ISSUE TITLE
MathJax needs to be uploaded to the website

### ISSUE THREAD

**Issue by retabak** [bootstrapworld/curriculum#963]
This is in the asciidoc:

`Check out the expression below. What do you think the answer is?  This math problem went viral on social media recently, with math teachers arguing about what the answer was! Why might they disagree on the solution?`

`++++`
`<style>`
`.centered-image.big, .centered-image.big p {margin-top: 0px; padding-top: 0px;}`
`.big .MathJax {font-size: 4em; color: black;}`
`</style>`
`++++`
`[.centered-image.big]`
`@math{6 \div 2(1 + 2)}`

`Instead of using a rule for computing answers, let's start by diagramming the math itself!`

This is what I see when I build:
![image](https://user-images.githubusercontent.com/87144723/174137495-f8af9cc6-6161-4c68-a1e8-3b90b6ddcdd9.png)

This is what I see at https://bootstrapworld.org/materials/fall2022-beta/en-us/lessons/order-of-operations/index.shtml?pathway=algebra-pyret :

![image](https://user-images.githubusercontent.com/87144723/174137628-1fac2660-2c53-4c77-953d-8102e0424a99.png)

**Comment by retabak** [comment]
Hm, same issue here (same lesson):

![image](https://user-images.githubusercontent.com/87144723/174138138-8eea9392-cdd5-4ab2-a596-feed3225df5a.png)

**Comment by schanzer** [comment]
@retabak apparently a lesson isn't being added to the dependency graph, which is causing all the javascript on the page to halt before finishing. This kills ALL mathjax.

**Comment by schanzer** [comment]
@retabak actually, my bad -- it looks like MathJax was not uploaded as part of the deployment script! @ds26gte the `extlib` folder should be uploaded, no?

**Comment by schanzer** [comment]
Fixed!
